client/src/app/Main.res

```diff
@@ -1975,7 +1970,7 @@ let update_ = (msg: msg, m: model): modification => {
   | GoToArchitecturalView =>
     Many(list{
       ReplaceAllModificationsWithThisOne(
-        m => ({...m, canvasProps: {...m.canvasProps, minimap: None}}, Cmd.none),
+        m => ({...m, canvasProps: m.canvasProps,}, Cmd.none),
```

Comment:
```suggestion
        m => (m, Cmd.none),
```

Reply:
Ah, good point.

That leaves the code like this:
```
| GoToArchitecturalView =>
    Many(list{
      ReplaceAllModificationsWithThisOne(
        m => (m, Cmd.none),
      ),
      Deselect,
      MakeCmd(Url.navigateTo(Architecture)),
    })
```

It seems like I can remove the `ReplaceAllModificationsWithThisOne` element of that list, then?

Reply:
Maybe this is wrong - could be some sort of 'reset pending modifications' that I don't understand.

- [ ] figure out what `ReplaceAllModificationsWithThisOne` is - it seems a bit scattered in code.

Reply:
after more inspection, I'm going to remove it - seems useless.